Core/GDCore/Project/MeasurementUnit.cpp

```diff
@@ -22,6 +22,49 @@ gd::MeasurementUnit MeasurementUnit::pixelAcceleration =
     CreatePixelAcceleration();
 gd::MeasurementUnit MeasurementUnit::newton = CreateNewton();
 gd::MeasurementUnit MeasurementUnit::angularSpeed = CreateAngularSpeed();
+std::vector<gd::MeasurementUnit*> MeasurementUnit::defaultMeasurementUnits;
+
+const std::vector<gd::MeasurementUnit*> &
+MeasurementUnit::GetDefaultMeasurementUnits() {
+  if (defaultMeasurementUnits.size() == 0) {
+    defaultMeasurementUnits.push_back(&undefined);
+    defaultMeasurementUnits.push_back(&dimensionless);
+    defaultMeasurementUnits.push_back(&degreeAngle);
+    defaultMeasurementUnits.push_back(&second);
+    defaultMeasurementUnits.push_back(&pixel);
+    defaultMeasurementUnits.push_back(&pixelSpeed);
+    defaultMeasurementUnits.push_back(&pixelAcceleration);
+    defaultMeasurementUnits.push_back(&angularSpeed);
+    defaultMeasurementUnits.push_back(&newton);
+  }
+  return defaultMeasurementUnits;
+}
+
+std::size_t MeasurementUnit::GetDefaultMeasurementUnitCount() {
```

Comment:
should this be `GetDefaultMeasurementUnitsCount` (with an s)

Reply:
+1